filequeue/filequeue_test.go

```diff
@@ -11,23 +11,19 @@ import (
 	"github.com/go-kit/log"
 	"github.com/grafana/walqueue/types"
 	"github.com/stretchr/testify/require"
-	"github.com/vladopajic/go-actor/actor"
-	"go.uber.org/goleak"
 )
 
 func TestFileQueue(t *testing.T) {
-	defer goleak.VerifyNone(t)
```

Comment:
Since we no longer explicitly close the threads this is no longer needed since there will always be a dangling channel.